dom.bs

```diff
@@ -744,11 +744,16 @@ steps:
 
  <li>
   <p>For each <var>tuple</var> in <a>context object</a>'s <a for=Event>path</a>:
-
-  <ol><li><p>If <var>currentTarget</var> is a <a>node</a> and <var>tuple</var>'s <b>item</b> is not
-  <a>closed-shadow-hidden</a> from <var>currentTarget</var>, or <var>currentTarget</var> is not a
-  <a>node</a>, then append <var>tuple</var>'s <b>item</b> to <var>composedPath</var>.</p></li></ol>
-
+  <ol>
+   <li><p>If <var>currentTarget</var> is a {{Window}}, then:
+    <ol><li><p>If <var>tuple</var>'s <b>item</b> is not a <a>node</a>, or <var>tuple</var>'s
+    <b>item</b> is not <a>closed-shadow-hidden</a> from <var>tuple</var>'s <b>item</b>'s <a>node
+    document</a>, then append <var>tuple</var>'s <b>item</b> to <var>composedPath</var>.</ol>
```

Comment:
Why node document? Would shadow-including root not be a little cleaner if that's what you're going for? (I guess there's maybe a difference too, if the node is not currently in a document, although that would not yield a different result I think.)